src/eq_schema/schema/Question/index.js

```diff
@@ -59,7 +60,40 @@ class Question {
     const dateRange = findDateRange(question);
     const mutuallyExclusive = findMutuallyExclusive(question);
 
-    if (dateRange) {
+    if (question.answers.some((answer) => answer.repeatingLabelAndInput)) {
+      this.type = "General";
+      this.dynamic_answers = {
+        values: {
+          source: "list",
+          identifier: getList(
+            ctx,
+            question.answers[0].repeatingLabelAndInputListId
+          ).listName,
+        },
+
+        answers: this.buildAnswers(question.answers, ctx),
+      };
+
+      if (question.totalValidation && question.totalValidation.enabled) {
+        this.type = "Calculated";
+        this.calculations = question.totalValidation.allowUnanswered
+          ? [
+              this.buildUnansweredCalculation(question.answers),
+              this.buildCalculation(
+                question.totalValidation,
+                question.answers,
+                ctx
+              ),
+            ]
+          : [
+              this.buildCalculation(
+                question.totalValidation,
+                question.answers,
+                ctx
+              ),
+            ];
+      }
+    } else if (dateRange) {
```

Comment:
Is the else necessary? can it just be an if statement? Basically, if theres a repeating answer then there won't be a date range.

Reply:
If the else is excluded then lines 157 to 160 will be executed, which would add the answers array below the question level

Reply:
ah ok